sql/moz-fx-data-shared-prod/search_terms_derived/sanitation_job_metadata/init.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE IF NOT EXISTS
+  `moz-fx-data-shared-prod.search_terms_derived.sanitation_job_metadata`
+AS
```

Comment:
You should include in the init.sql query the options that are consistent with the metadata e.g.

```
PARTITION BY
  DATE(submission_timestamp)
```

```
OPTIONS
  (require_partition_filter = TRUE, partition_expiration_days = 15)
```